custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -141,7 +142,7 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
         {id: 10, name: 'Take Home Ration (THR)'},
     ];
 
-    if (vm.haveAccessToFeatures) {
+    if (vm.haveAccessToFeatures && vm.userLocationType.toLowerCase() !== 'block') {
```

Comment:
Had a question on the ticket about whether this logic is correct or it should actually be any user below the block level...